docs/quickstart/quickstart-non-docker.md

```diff
@@ -57,6 +57,10 @@ connect is [UP]
     $ mvn clean compile install -DskipTests
     ```
 
+    > Tip:  When using a Maven proxy/mirror in your company such as Artifactory you must make sure that it is
```

Comment:
i'd revise to: `Tip:  When using a Maven proxy or mirror, such as Artifactory, make sure that it is correctly configured to retrieve snapshot artifacts and access snapshot repositories.  For more information, see [Could not find artifact](https://github.com/confluentinc/ksql/issues/537). `